src/features/party/PartiesProvider.tsx

```diff
@@ -196,3 +207,16 @@ function getCookieValue(name: string): string | null {
       ?.split('=')[1] ?? null
   );
 }
+
+/**
+ * The party from the profile or a the system user party.
+ * If there is no profile, the user is a system user.
```

Comment:
This goes for service owners also currently. Service owners and fill out and proces/next a data-step, and they also don't have a profile